core/state_processor.go

```diff
@@ -126,9 +126,9 @@ func applyTransaction(msg types.Message, config *params.ChainConfig, bc ChainCon
 	}
 
 	// Set the receipt logs and create the bloom filter.
-	receipt.Logs = statedb.GetLogs(tx.Hash())
+	receipt.Logs = statedb.GetLogs(tx.Hash(), header.Hash())
```

Comment:
reuse it, a header does not store the hash, as opposed to a block

Reply:
I mean two lines below, you can reuse the hashing you did here

Reply:
done